SmplEditor/Smpl.cs

```diff
@@ -27,8 +27,33 @@ internal class Smpl
             this.sortBy = sortBy;
             this.version = version;
         }
-        public Smpl(string name, List<SmplSong> songList){
-            ;
+        public Smpl(Playlist playlist){
```

Comment:
Add description with reasoning of why this seemingly "reverse" constructor is required.
--> Exporting.
Importing requires different type of playlists to be converted to a standardized type `Song`.
To export, it makes sense that they are reverse converted.